packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -488,7 +496,27 @@
   }
 
   .is-calendar-focused {
-    outline: var(--terra-date-picker-month-focus-outline, 2px dashed #000);
-    outline-offset: var(--terra-date-picker-month-focus-outline-offset, 0);
+    &--keyboard-focus {
+      outline: var(--terra-date-picker-day-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-date-picker-day-focus-outline-offset, 0);
+    }
+  }
+
+  .react-datepicker-day {
+    &--selected-border {
+      outline: var(--terra-date-picker-day-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-date-picker-day-focus-outline-offset, 0);
```

Comment:
Should these theme variables be provided with values for fusion or lowlight themes?

Reply:
I did not see the values for the previous variable also which was renamed in the theme files.

Reply:
@jeremyfuksa Now that we have dashed borders for the focus style, should the fusion and lowlight themes use different colors for the border?

Reply:
I had to dig around Zeplin to find examples of what the focus style should be.

I didn't find any examples for Fusion, so I'm going to make the call that it's OK to have focus color on open source and Fusion to be the same. Low Light shows that the border color is the same as its text color - #1078 B7B7B8

Reply:
I am seeing this [comment](https://github.com/cerner/terra-framework/blob/66db72147831e2093b7a876a9db3e1b9d3d42276/packages/terra-date-picker/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss#L29) in the theme file so added the day outline variable in this list.